src/explore-education-statistics-common/src/components/ChartsToPNG.tsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { useExportButtonContext } from '@common/contexts/ExportButtonContext';
+import html2canvas, { Options as HTML2CanvasOptions } from 'html2canvas';
+import { useCallback, useState } from 'react';
+
+export type UseGenerateImage<T extends HTMLElement = HTMLDivElement> = [
```

Comment:
As with `UseCurrentPng` this doesn't need to be exported and might be better named `UseGenerateImageType`.

Is the `T` doing anything? My editor says it's declared but not read.